distribution/dashboard/internal/router/v1/obtenant_router.go

```diff
@@ -9,6 +9,19 @@ func InitOBTenantRoutes(g *gin.RouterGroup) {
 	g.GET("/obtenants", handler.ListAllTenants)
 	g.GET("/obtenant/:namespace/:name", handler.GetTenant)
 	g.PUT("/obtenant/:namespace/:name", handler.CreateTenant)
-	g.POST("/obtenant/:namespace/:name", handler.UpdateTenant)
 	g.DELETE("/obtenant/:namespace/:name", handler.DeleteTenant)
+
+	g.PUT("/obtenant/:namespace/:name/unitNumber", handler.ModifyUnitNumber)
```

Comment:
why not using patch for modification
ob-operator supports modify multiple configurations